source/tyr/tyr/resources.py

```diff
@@ -1190,13 +1190,15 @@ def post(self, user_id, version=0):
 
 
 class EndPoint(flask_restful.Resource):
-    @marshal_with(end_point_fields)
-    def get(self):
-        return models.EndPoint.query.all()
+    def get(self, version=0):
+        resp = marshal(models.EndPoint.query.all(), end_point_fields)
+        if version == 1:
+            return {'end_points': resp}
+        return resp
```

Comment:
(i know it may be overkill for this PR, it depends on if you want to organize v0 and v1 better)

It seems that we're mixing functional codes and serialization code according to the version of api. 

in order to de-correlate these logics, can we, for every endpoint,  do something like,
```python
return serializer_select(version).serialize(endpoint_impl_func(...))
```

Reply:
I created a ticket to monitor the upgrade of the API v1, which, I fully agree, is optimized as is:
https://jira.kisio.org/browse/NAVP-1409